nservicebus/upgrades/5to6-moving-away-from-ibus.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Moving away from `IBus` in NServiceBus Version 6
+summary: Describes how to send messages without the IBus
+---
+
+Starting with Versions 6 and above, the `IBus` interface has been deprecated.
+
+The `IBus` interface was one of the key interfaces when using previous versions of NServiceBus. It provided access to many of the important operations like sending messages, subscribing to events, and manipulating headers. The `IBus` interface was injected into the container using dependency injection and was accessible in all the message handlers and other injected components in the container as well. 
+
+There are several scenarios when using IBus:
```

Comment:
I've now changed it to:

Some scenarios involving `IBus` include endpoint creation, sending messages during endpoint startup, sending messages from within message handlers, using the injected `IBus` in custom components also registered in the container.